### ISSUE TITLE
Restyler Job #410154 fails with empty log

### ISSUE THREAD

**Issue by Damian-Nordic** [restyled-io/restyler#127]
Hi there-

I'm having a problem with a Restyled Job

- **Restyled Job**: https://restyled.io/gh/project-chip/repos/connectedhomeip/jobs/410154
- **GitHub PR**: project-chip/connectedhomeip#3303

**What I expected to happen**:

I expected restyler to complete the job or report a meaningful error.

**What happened instead**:

I got information that the job failed, but no message was printed to the log.

**Configuration** (if applicable):

```yaml
# .restyled.yaml as it was in the commit for the Job
```

**Comment by pbrisbin** [comment]
Thank you for reporting this. I've been battling [an infrastructure issue](https://github.com/restyled-io/ops/issues/7) since yesterday and your job was one of the ones that errored as a result. The issue is not occurring at the moment, but I'm still working on fixes to prevent it from re-occurring.

It may take some time, but I hope to:

- Make the system robust to this particular failure going forward
- Make it so these kind of unknown-unknowns still always result in _something_ in the Build logs

Sorry for the inconvenience in the meantime.

**Comment by Damian-Nordic** [comment]
@pbrisbin No problem, the job has passed today. Thanks for the quick response!
